packages/core/upload/admin/src/components/FolderCard/FolderCard/FolderCard.js

```diff
@@ -83,22 +95,26 @@ export const FolderCard = ({
 
           {children}
 
-          {cardActions && <CardAction right={4}>{cardActions}</CardAction>}
+          <CardActionDisplay zIndex={3}>
```

Comment:
Who do you need to use `3` instead of `1` or `2`?

Reply:
good question - I didn't find why exactly but if I put 1 or 2 the IconButton stays under the text and then is not clickable 🤔

<img width="222" alt="image" src="https://user-images.githubusercontent.com/71838159/170009631-8c9a172f-4b72-4885-b6b1-4ed06a694d4d.png">

Reply:
found out that the `FolderCardBody` had a zIndex prop that caused this - I fixed it!